BUILDING_ON_WINDOWS.md

```diff
@@ -28,15 +29,26 @@ Notes:
 #### When prompted which components to install:
 
 1. Unfold the tree element that says "Qt"
-2. Unfold the top most tree element (latest stable Qt version, e.g. `Qt 5.15.2`)
+2. Unfold the top most tree element (latest stable Qt version, e.g. `Qt 6.5.3`)
 3. Under this version, select the following entries:
    - `MSVC 2019 64-bit` (or alternative version if you are using that)
 4. Under the "Tools" tree element (at the bottom), ensure that `Qt Creator X.X.X` and `Debugging Tools for Windows` are selected. (they should be checked by default)
 5. Continue through the installer and let the installer finish installing Qt.
 
 Note: This installation will take about 2 GB of disk space.
 
-Once Qt is done installing, make sure you add its bin directory to your `PATH` (e.g. `C:\Qt\5.15.2\msvc2019_64\bin`)
+Once Qt is done installing, make sure you add its bin directory to your `PATH` (e.g. `C:\Qt\6.5.3\msvc2019_64\bin`)
+
+<details>
+   <summary>How to add Qt to PATH</summary>
+   
+1. Type "path" in the Windows start menu and click `Edit the system environment variables`.
+2. Click the `Environment Variables...` button bottom right.
+3. In the `System variables` section, scroll down until you find `Path` and double click it.
```

Comment:
This is personal preference: Qt doesn't have to be added for all users (system-wide), so the `User variables` are fine as well.

```suggestion
3. In the `User variables` (scoped to the current user) or `System variables` (system-wide) section, scroll down until you find `Path` and double click it.
```